modules/datadict/README.md

```diff
@@ -0,0 +1,51 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary descriptions.
+
+## Scope
+
+The module displays and manages the data dictionary for things
+already stored in the LORIS `parameter_type` table. This table is
+generally autopopulated and provides a dictionary for imaging headers
+(populated by the imaging pipeline) and behavioural instrument
+(populated by the `lorisform_parser.php` script).
+
+It does not provide a way to enter new things into the data dictionary,
```

Comment:
Maybe replace "things" by "fields"?

Reply:
sorry for the delay.. with the changes.. done. (And also one instance in the paragraph above your comment)